### PR TITLE
e2e: use interchaintest main

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

[This](https://github.com/strangelove-ventures/interchaintest/pull/741) should get us out of @colin-axner's fork.

closes: #XXXX

### Commit Message / Changelog Entry

```bash
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

**Review by damiannolan — APPROVED** [review]

**Comment by damiannolan** [comment]
e2e build is failing actually, looks like something to do with the `UseNewGenesisCmd` flag

**Comment by crodriguezvega** [comment]
hmm...

```
testsuite/testconfig.go:427:3: unknown field UsingNewGenesisCommand in struct literal of type "github.com/strangelove-ventures/interchaintest/v8/ibc".ChainConfig
```

Seems like something else broke... I will take a look later.

**Comment by damiannolan** [comment]
It was removed in https://github.com/strangelove-ventures/interchaintest/pull/759 and looks like its determined internally now

File: e2e/go.mod [hunk 1]
```diff
@@ -236,5 +236,3 @@ replace github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.2021
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 //replace github.com/strangelove-ventures/interchaintest/v8 => github.com/notional-labs/interchaintest/v8 v8.0.0-20230905211440-c2dc240d2221
```
Comment (colin-axner) [inline e2e/go.mod:238]:
```suggestion
```

File: e2e/testsuite/testconfig.go [hunk 2]
```diff
@@ -413,18 +404,17 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string, cometCf
...
-		CoinType:               fmt.Sprint(sdk.GetConfig().GetCoinType()),
-		Denom:                  denom,
-		EncodingConfig:         SDKEncodingConfig(),
-		GasPrices:              fmt.Sprintf("0.00%s", denom),
-		GasAdjustment:          1.3,
-		TrustingPeriod:         "508h",
-		NoHostMount:            false,
-		ModifyGenesis:          getGenesisModificationFunction(cc),
-		ConfigFileOverrides:    configFileOverrides,
-		UsingNewGenesisCommand: useNewGenesisCommand,
```
Comment (colin-axner) [inline e2e/testsuite/testconfig.go:427]: how is backwards compatibility being handled now?

**Review by colin-axner — APPROVED** [review]
ACK

File: e2e/testsuite/testconfig.go (hunk 2 above)
Comment (damiannolan) [reply e2e/testsuite/testconfig.go:427]: had the same thoughts wrt backwards compat, but fingers crossed the internal lookup in interchaintest handles this across the board

File: e2e/tests/upgrades/genesis_test.go [hunk 3]
```diff
@@ -198,12 +197,6 @@ func (s *GenesisTestSuite) HaltChainAndExportGenesis(ctx context.Context, chain
 	state, err := chain.ExportState(ctx, int64(haltHeight))
 	s.Require().NoError(err)
 
-	// state exports currently read from stdout which include log entries, this discards the log entry by the server module
-	// this may need to be updated if log entries are inserted in between exporting of state and the server module logging
-	// see issue: https://github.com/strangelove-ventures/interchaintest/issues/721
-	str := strings.SplitAfter(state, "server")
-	state = str[1]
```
Comment (crodriguezvega) [inline e2e/tests/upgrades/genesis_test.go:205]: The [issue](https://github.com/strangelove-ventures/interchaintest/issues/721) referenced here has been closed.

**Review by DimitrisJim — APPROVED** [review]

**Pull request merged**